superwall/src/main/java/com/superwall/sdk/network/device/DeviceHelper.kt

```diff
@@ -467,6 +470,8 @@ class DeviceHelper(
                 ipCity = geo?.city,
                 ipContinent = geo?.continent,
                 ipTimezone = geo?.timezone,
+                capabilities = capabilities.map { it.name },
```

Comment:
Is this comma separated?

Reply:
Yup, when toString is called on this default implementation is comma separated 👍